components/sysutils/ngrep/Makefile

```diff
@@ -10,13 +10,16 @@
 
 #
 # Copyright 2016 Adam Stevko 
+# Copyright 2023 Klaus Ziegler
 #
 
+BUILD_BITS= 64
```

Comment:
No need to specify this, 64bit is the default (see 26233532a668304e19972ecf3be66aa929a97e13)